make/migrations/postgresql/0004_add_robot_account.up.sql

```diff
@@ -3,6 +3,7 @@ CREATE TABLE robot (
  name varchar(255),
  description varchar(1024),
  project_id int,
+ expiration int,
```

Comment:
https://github.com/dgrijalva/jwt-go/blob/master/claims.go#L20
This is a `int64`, so probably we should use `bigint`